src/olympia/lib/settings_base.py

```diff
@@ -477,6 +477,8 @@ def lazy_langs(languages):
 ROOT_URLCONF = 'olympia.urls'
 
 INSTALLED_APPS = (
+    'olympia.translations',
```

Comment:
I don't remember the reason for this, needs investigation and a comment I guess. Maybe that was experimental? :thinking: